app/Domain/Integrations/Controllers/IntegrationController.php

```diff
@@ -151,45 +212,39 @@ public function deleteContact(Request $request, string $id, string $contactId):
         }
 
         return Redirect::route(
-            TranslatedRoute::getTranslatedRouteName(
-                request: $request,
-                routeName: 'integrations.detail'
-            ),
+            TranslatedRoute::getTranslatedRouteName($request, 'integrations.show'),
             [
                 'id' => $id,
             ]
         );
     }
 
-    public function updateBilling(string $id, UpdateBillingInfo $updateBillingInfo): RedirectResponse
+    public function updateBilling(string $id, UpdateBillingInfoRequest $request): RedirectResponse
     {
         $organisation = new Organization(
-            Uuid::fromString($updateBillingInfo->input('organisation.id')),
-            $updateBillingInfo->input('organisation.name'),
+            Uuid::fromString($request->input('organisation.id')),
+            $request->input('organisation.name'),
             'test@test.be',
```

Comment:
Is this `test@test.be` intentional or a typo?

Reply:
A remaining dummy value, will fix it